apolloschurchapp/src/tabs/index.js

```diff
@@ -111,20 +104,46 @@ const HomeTab = createFeatureFeedTab({
 });
 
 const ReadTab = createFeatureFeedTab({
+  screenOptions: {
```

Comment:
So this is a small thing, but this should be:

```suggestion
  options: {
```

The difference is that `screenOptions` seems to override not only this screen, but any children. So like if you have a "View All" button on the tab that opens up a nested route - the user profile avatar is still visible. Using `options` means that it's only applied to the header on this tab, and if you navigate to a child route you'll see a back button!

I did it this way [on Chase Oaks](https://github.com/Differential/chase-oaks-apollos/blob/master/apolloschurchapp/src/tabs/index.js#L96) after experimenting with both.

Reply:
oh nice